tests/config/config-export.spec.ts

```diff
@@ -393,6 +393,87 @@ describe("Config export", () => {
         expect(secondPackageContent.variables).toEqual([...secondPackageVariableDefinition]);
     })
 
+    it("Should export with SCENARIO nodes removed and CONNECTION variables fixed for package key with multiple underscores", async () => {
+        const manifest: PackageManifestTransport[] = [];
```

Comment:
Do we have a test for types of keys that don't have any underscores?

Reply:
Yes, all other tests are with keys without underscores.